firebase-inappmessaging/src/test/java/com/google/firebase/inappmessaging/internal/InAppMessageStreamManagerTest.java

```diff
@@ -560,7 +559,7 @@ public void stream_onCacheWriteFailure_AbsorbsError() {
 
   @Test
   public void stream_whenCampaignImpressed_filtersCampaign() {
-    when(impressionStorageClient.isImpressed(anyString())).thenReturn(Single.just(true));
+    when(impressionStorageClient.isImpressed(any())).thenReturn(Single.just(true));
```

Comment:
why did we change this? The more specific it is, the better. I'd vote for bringing this back.